docs/release-notes/cloud/cloud-2023-01.md

```diff
@@ -0,0 +1,88 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud January 2023.
+included_jira_versions: ['2023.01']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.7.36
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.7.76
+---
+
+# Cloud January 2023
+
+These release notes are for the Codacy Cloud updates during January 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The Codacy Coverage Reporter now supports [automatic commit SHA-1 detection](../../coverage-reporter/troubleshooting-coverage-cli-issues.md#commit-detection) on Argo CD. (TS-140)
+-   Extended the visibility of the [coding standards page](../../organizations/using-a-coding-standard.md) to all users, regardless of their permission level. (IO-359)
```

Comment:
@andre-a-pires Please validate that this update is good for including on our January release notes.

Reply:
Resolving this one to merge. @andre-a-pires feel free to ping any of us tech writers if needed!